langs/ptbr.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Português-Brasil",
```

```suggestion
  "name": "Português Brasileiro",
```
It's the official name for the language.